### PR TITLE
fixed #9801 filters overlapping to headers issue

### LINKED ISSUE #9801: In Category Page -> Filters are Overlapping into the header.
### Terms

- [X] Before you openning this issue, i have checked if the issue has already been reported.

### Bagisto Version(s) affected

Master

### Issue Description

When we were checking the filter and there we encounter this issue in which Category Dropdown filter is Overlapping with the header where the categories are present.

### Preconditions

1. Admin must be logged in.
2. Categories must be created.

### Steps to reproduce

1. Go to Shop Front -> Select Catgory -> Open the dropdown filter and Scroll the page.
2. There we can see that it is overlapping with the Header where categories are present.

### Expected Result

It should not overlap the category Header.

### Actual Result

Currently we can see that the flters are overlapping . I have checked and verified this issue in RTL and LTR view. Please check the video for reference.

**Video**

https://github.com/bagisto/bagisto/assets/92721837/c70d0bf4-6761-4a85-81ce-2906b164fc02

### Screenshots

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by hussaincoder23**
## Issue Reference
fixed #9801 filters overlapping to headers issue
## Description
I have fixed issue where filters are overlapping to the headers

### CODE REVIEW

**Comment by ashishkumar-webkul** [comment]
@hussaincoder23 
Thank you so much for your contribution. We really appreciate your efforts. After checking and testing the PR, we found that the locale dropdown top 2 locales are getting hidden behind the header. Kindly check  the video for reference, we would really happy if  you could fix the issue as well.

Thanks 

**Video**

https://github.com/bagisto/bagisto/assets/92721837/ecc596bf-62da-4fa3-93b8-24e4e991882a

**Comment by hussaincoder23** [comment]
Ok fixing that locale dropdown issue as well @ashishkumar-webkul

**Comment by suraj-webkul** [comment]
@hussaincoder23

Thank you so much for your contribution. We really appreciate your efforts and dedication.

However, we are currently avoiding the use of inline CSS as we are using Tailwind CSS. Please adjust the code accordingly.

**Comment by hussaincoder23** [comment]
@suraj-webkul  updated PR and used only tailwind class .

**Comment by ashishkumar-webkul** [comment]
@hussaincoder23 
Please check there are some conflicts in the PR, please resolve the conflicts.

**Comment by hussaincoder23** [comment]
@ashishkumar-webkul  fixed language locale dropdown hidden behind header issue

**Comment by ashishkumar-webkul** [comment]
@hussaincoder23 

As I have checked and verified the issue, now it is working fine. please check  the video for reference. 

**Video**

https://github.com/bagisto/bagisto/assets/92721837/18e8e8e8-5e86-4ced-baed-6e5057b0c0e5

Thank You So Much for your contribution, looking ahead for more.

File: packages/Webkul/Shop/src/Resources/views/categories/toolbar.blade.php [hunk 1]
```diff
@@ -17,7 +17,7 @@
                 {!! view_render_event('bagisto.shop.categories.toolbar.filter.before') !!}
 
                 <!-- Product Sorting Filters -->
-                <x-shop::dropdown position="bottom-left">
+                <x-shop::dropdown class="z-[1]" position="bottom-left">
```
Comment (jitendra-webkul) [inline packages/Webkul/Shop/src/Resources/views/categories/toolbar.blade.php:20]: To improve readability, I suggest breaking down your code into multiple lines. If a tag has more than one attribute, it should be formatted over multiple lines

Comment (hussaincoder23) [reply packages/Webkul/Shop/src/Resources/views/categories/toolbar.blade.php]: Yupp , code updated as per change request .

File: packages/Webkul/Shop/src/Resources/views/categories/toolbar.blade.php [hunk 2]
```diff
@@ -17,7 +17,10 @@
                 {!! view_render_event('bagisto.shop.categories.toolbar.filter.before') !!}
 
                 <!-- Product Sorting Filters -->
-                <x-shop::dropdown position="bottom-left">
+                <x-shop::dropdown 
+                        class="z-[1]" 
+                        position="bottom-left"
+                        >
```
Comment (shivendra-webkul) [inline packages/Webkul/Shop/src/Resources/views/categories/toolbar.blade.php:23]: To improve readability fix the extra space

Comment (hussaincoder23) [reply packages/Webkul/Shop/src/Resources/views/categories/toolbar.blade.php]: is it okay now ? @shivendra-webkul

Comment (shivendra-webkul) [reply packages/Webkul/Shop/src/Resources/views/categories/toolbar.blade.php]: Remove the space before the ">" in the tag.

It should be something like this 

Reference :

<x-shop::dropdown 
    class="z-[1]" 
    position="bottom-left"
>

Comment (shivendra-webkul) [reply packages/Webkul/Shop/src/Resources/views/categories/toolbar.blade.php]: @hussaincoder23, Thank you for your contribution.

**Review by amit-webkul — APPROVED** [review]

**Review by shivendra-webkul — APPROVED** [review]

**Comment by ashishkumar-webkul** [comment]
Checked and Found that the issue has been fixed now and working fine. Please check the video for reference.

**Video**

https://github.com/bagisto/bagisto/assets/92721837/ac04958d-897d-480c-9983-b34c46cec9b9

**Review by devansh-webkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**